.release/pypi/inference.core.setup.py

```diff
@@ -65,7 +65,6 @@ def read_requirements(path):
         "hosted": read_requirements("requirements/requirements.hosted.txt"),
         "http": read_requirements("requirements/requirements.http.txt"),
         "sam": read_requirements("requirements/requirements.sam.txt"),
-        "sam2": read_requirements("requirements/requirements.sam2.txt"),
```

Comment:
was that extras ever released?

Reply:
Yes as part of last release I think.  Probably not very many installs if any; but maybe its possible to leave it as empty?

Reply:
👍

Reply:
lets remove then